src/commcare_cloud/fab/fabfile.py

```diff
@@ -241,10 +241,7 @@ def send_email(subject, message, use_current_release=False):
 
 @task
 def kill_stale_celery_workers():
-    """
-    Kills celery workers that failed to properly go into warm shutdown
-    """
-    execute(release.kill_stale_celery_workers)
+    print("OBSOLETE use 'kill-stale-celery-workers' instead")
```

Comment:
Just thought of this, but if you put this message ("OBSOLETE ...") in the docstring as well, instead of removing it, then it would show up in the fab help output and documentation, which will go just a tad further into passively communicating the deprecation to people. Not something I'd insist on, just a passing thought.

Reply:
46233a411e34f5cbee01c71e19748c61c5ead34f